client/src/main/java/com/arcbees/beeshop/client/application/widget/sidepanel/cart/cartitem/CartItemUiHandlers.java

```diff
@@ -20,4 +20,6 @@
 
 public interface CartItemUiHandlers extends UiHandlers {
     void delete();
+
+    void onQuantityChangedInView(int quantity);
```

Comment:
I hate to add the "inView", but there's already a "onQuantityUpdated" method in the presenter (event handler), so it was too confusing.